addons/barcodes/static/src/barcode_handler_field.js

```diff
@@ -0,0 +1,24 @@
+/** @odoo-module **/
+
+import { registry } from "@web/core/registry";
+import { standardFieldProps } from "@web/views/fields/standard_field_props";
+import { useBus, useService } from "@web/core/utils/hooks";
+
+const { Component, xml } = owl;
+
+export class BarcodeHandlerField extends Component {
+    setup() {
+        super.setup();
```

Comment:
if you inherit from component, no need for super.setup(). Owl guarantees that it does not implement setup, it's only for user code